hpeOneView/resources/security/users.py

```diff
@@ -108,8 +108,148 @@ def get_by_userName(self, userName):
             uri = self.URI + '/role/' + userName
             self._helper.validate_resource_uri(uri)
             data = self._helper.do_get(uri)
-            new_resource = self.new(self._connection, data)
+            result = data['members']
+            return result
         else:
-            new_resource = None
+            return None
+
+    def get_by_userName(self, name):
+        """
+        Gets a complete json body for username
+
+        Args:
+          name: userName of the user
+
+        Returns:
+           dict: User
+        """
+
+        uri = self._helper.build_uri(name)
+        new_resource = self.get_by_uri(uri)
 
         return new_resource
+
+    def get_user_by_role(self, rolename):
+        """
+        Gets all the users associated with this role
+
+        Args:
+          rolename: rolename of the user
+
+        Returns:
+          dict: User
+        """
+
+        rolename = quote(rolename)
+        uri = self.URI + '/roles/users/' + rolename
+        data = self._helper.do_get(uri)
+        result = []
+        for i in range(0, len(data['members'])):
+            result.append(data["members"][i])
+
+        return result
+
+    def create_multiple_user(self, user):
+        """
+        Create a multiple user
+
+        Agrs:
+          user: multiple user
+
+        Returns:
+          dict: User
+        """
+
+        uri = self.URI + '?multiResource=true'
+        return self._helper.create(user, uri)
```

Comment:
try to use methods from ```Resource``` class instead of ```ResourceHelper``` class since you have already inherited the Resource class